hack/conformance/check_conformance_test_requirements.go

```diff
@@ -73,10 +72,10 @@ func checkAllProviders(e2eFile string) error {
 		}
 	}
 	if inConformanceCode {
-		return errors.Errorf("Missed the end of previous conformance test. There might be a bug in this script.")
+		return fmt.Errorf("Missed the end of previous conformance test. There might be a bug in this script.")
```

Comment:
nit: this should probably just be errors.New ? (the stdlib errors)
lots of unnecessary use of format string processing in this file 😅

Reply:
Thanks . I have used errors.New instead of fmt.Errorf.